### PR TITLE
Fix nytt medlem page

### PR DESCRIPTION
**Pull request by KWaloen**
Fixed and clarified text up to Andeby as thats as far as I got yesterday with the new member. Will update more once I go through it. Double check my Norwegian pls!

### CODE REVIEW

File: content/docs/innmelding/nytt_medlem.md [hunk 1]
```diff
@@ -25,16 +25,19 @@ For å lage en ny epost-adresse for det nye medlemmet.
...
-3. Gå til Mailboxes
-4. Velg fribyte@fribyte.no
-5. Under delegation: Legg til ny epost addresse i tekstfeltet for Local
+2. Gå til Mailboxes
+3. Gå til "List" -> "aktive@friByte.no"
+4. Under Overview -> Delegation
+5. Legg til ny epost addresse i tekstfeltet for Local
    Recipients
+6. Save Changes
+7. Even more Profitt
```
Comment (mathiash98) [inline content/docs/innmelding/nytt_medlem.md:40]: Stonks! 🤑 🚀

**Review by skivdal — COMMENTED** [review]
Veldig bra! Har pirket litt, men ellers er norsken helt perfekt :D

File: content/docs/innmelding/nytt_medlem.md [hunk 2]
```diff
@@ -58,12 +61,25 @@ Andeby gjennom [SSH-protokollen](https://www.ssh.com/academy/ssh).
...
+`ssh-keygen -t ed25519 -C "your_email@example.com"` for å generere en SSH-nøkkel. Få mer info:
    [Github docs, how to generate a new ssh key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
-2. Legg til den nye genererte nøkkelen til '~/.ssh/authorized_keys' i Bestemor.
-   Dette er noe et aktivt medlem må gjøre.
+
+2. Dette er noe et aktivt medlem må gjøre.
+   
+3. Hente .pub key fra det nye medlemmet, skriv ut public key med `cat .\publickey.pub` og sende det til deg i friByte chat. 
+
+4. Koble til bestemor via `ssh root@andeby.fribyte.no`. Dette er gatewayen til resten av våre kjære maskiner.
```
Comment (mathiash98) [inline content/docs/innmelding/nytt_medlem.md:75]:
```suggestion
2. Hente .pub key fra det nye medlemmet, skriv ut public key med `cat .ssh/*.pub` og sende det til aktivt medlem i friByte chat. 

3. Neste steg må et aktivt medlem i friByte som har tilgang til Andeby gjøre

4. Koble til bestemor via `ssh root@andeby.fribyte.no`. Dette er gatewayen til resten av våre kjære maskiner. 
```

**Review by mathiash98 — APPROVED** [review]
Helt konge!

File: content/docs/innmelding/nytt_medlem.md [hunk 3]
```diff
@@ -58,12 +61,25 @@ Andeby gjennom [SSH-protokollen](https://www.ssh.com/academy/ssh).
...
 1. Om det nye medlemmet ikke har en SSH-nøkkel, må det genereres.
+
+`ssh-keygen -t ed25519 -C "your_email@example.com"` for å generere en SSH-nøkkel. Få mer info:
    [Github docs, how to generate a new ssh key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
-2. Legg til den nye genererte nøkkelen til '~/.ssh/authorized_keys' i Bestemor.
-   Dette er noe et aktivt medlem må gjøre.
+
+2. Dette er noe et aktivt medlem må gjøre.
+   
+3. Hente .pub key fra det nye medlemmet, skriv ut public key med `cat .\publickey.pub` og sende det til deg i friByte chat.
```
[the remark this answers is not in the document]
Comment (skivdal) [reply content/docs/innmelding/nytt_medlem.md:73]: Foretrekker Mathias sin løsning lengre nede til min :)

File: content/docs/innmelding/nytt_medlem.md [hunk 4]
```diff
@@ -58,12 +61,25 @@ Andeby gjennom [SSH-protokollen](https://www.ssh.com/academy/ssh).
...
-2. Legg til den nye genererte nøkkelen til '~/.ssh/authorized_keys' i Bestemor.
-   Dette er noe et aktivt medlem må gjøre.
+
+2. Dette er noe et aktivt medlem må gjøre.
+   
+3. Hente .pub key fra det nye medlemmet, skriv ut public key med `cat .\publickey.pub` og sende det til deg i friByte chat. 
+
+4. Koble til bestemor via `ssh root@andeby.fribyte.no`. Dette er gatewayen til resten av våre kjære maskiner. 
+
+5. Åpne authorized_keys i ditt favoritt text editor, nano for eksempel: `nano ~/.ssh/authorized_keys`.
```
[the remark this answers is not in the document]
Comment (Cengelsen) [reply content/docs/innmelding/nytt_medlem.md:77]: "kode-redaktør"

File: content/docs/innmelding/nytt_medlem.md (hunk 1 above)
Comment (Cengelsen) [inline content/docs/innmelding/nytt_medlem.md:40]:
```suggestion
6. Lagre endringer
7. Enda mer profitt
```

File: content/docs/innmelding/nytt_medlem.md [hunk 5]
```diff
@@ -58,12 +61,25 @@ Andeby gjennom [SSH-protokollen](https://www.ssh.com/academy/ssh).
...
+
+2. Dette er noe et aktivt medlem må gjøre.
+   
+3. Hente .pub key fra det nye medlemmet, skriv ut public key med `cat .\publickey.pub` og sende det til deg i friByte chat. 
+
+4. Koble til bestemor via `ssh root@andeby.fribyte.no`. Dette er gatewayen til resten av våre kjære maskiner. 
+
+5. Åpne authorized_keys i ditt favoritt text editor, nano for eksempel: `nano ~/.ssh/authorized_keys`. 
+
+6. Skriv navnet til det nye medlemmet og paste inn ssh key.
```
Comment (Cengelsen) [inline content/docs/innmelding/nytt_medlem.md:79]:
```suggestion
6. Skriv navnet til det nye medlemmet og lim inn ssh-nøkkelen. 
```

File: content/docs/innmelding/nytt_medlem.md [hunk 6]
```diff
@@ -25,16 +25,19 @@ For å lage en ny epost-adresse for det nye medlemmet.
 2. Gå til "Email Domains" -> "friByte.no"
 3. Under "All Addresses" eller "Mailboxes", trykk på "New Mailbox"
 4. Fyll inn medlemmets info
-5. Profitt
+5. Create Mailbox
```
Comment (Cengelsen) [inline content/docs/innmelding/nytt_medlem.md:28]:
```suggestion
5. Trykk "Create Mailbox"
```

**Review by Cengelsen — APPROVED** [review]

File: content/docs/innmelding/nytt_medlem.md (hunk 1 above)
Comment (KWaloen) [reply content/docs/innmelding/nytt_medlem.md]: 💯

**Pull request merged**